core/package.json

```diff
@@ -27,7 +27,8 @@
     "better-sqlite3": "^7.4.1",
     "moment": "^2.24.1",
     "uuid": "^8.3.2",
-    "ws": "^7.4.4"
+    "ws": "^7.4.4",
+    "devtools-protocol": "^0.0.799653"
```

Comment:
FWIW, I have devtools protocol exposed from the IDevtoolsSession interface. I did that to try to consolidate our references to devtools-protocol to a single place. Not super important if you don't want to use that.

Reply:
Cool. Removed the dependency. Pushed.